narwhals/_utils.py

```diff
@@ -2035,3 +2036,62 @@ def deep_attrgetter(attr: str, *nested: str) -> attrgetter[Any]:
 def deep_getattr(obj: Any, name_1: str, *nested: str) -> Any:
     """Perform a nested attribute lookup on `obj`."""
     return deep_attrgetter(name_1, *nested)(obj)
+
+
+def _is_close_impl(
```

Comment:
In principle, we could use this implementation also for polars pre 1.32.0, however polars has a couple of native methods which play a bit nicer (e.g. `.sign`, `.is_infinite`, `.not_`). If we were to introduce those, then we can have a single implementation function